### PR TITLE
Validation (112362375)

### PR DESCRIPTION
**Pull request by samuelmi2**
Added validate script, Makefile target and instance-module enhancement, also corrected a small error in README.

### CODE REVIEW

**Comment by samuelmi2** [comment]
I've signed it.

File: validate.sh [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+#}
+#auditor ""
+# TEST CASES:
+#a) confirming that the "owner" can create/view all resources - X
+#b) confirming that the "auditor" cannot list all pods, but CAN list pods in the dev namespace - X
+#c) confirming that the "auditor" cannot create or delete pods - X
+#d) deploying the pod-labeler application and confirming the labels appear on the pods - 
+#e) Makefile needs validate target
+
+owner() {
```
Comment (techgnosis) [inline validate.sh]: These three functions are defined identically in two different places. Can they be put into a single place and sourced?

File: validate.sh [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+# bash "strict-mode", fail immediately if there is a problem
+set -euo pipefail
+
+SUCCESS="hello-server-"
+FAILURE="Error from server (Forbidden)"
+UPDATED="updated="
+OUTPUT=$SUCCESS
+
+#auditor() {
```
Comment (techgnosis) [inline validate.sh]: This comment block is confusing. It looks like it starts off as old code then goes into auditor test cases, but does not discuss owner or admin test cases. Can this be cleaned up?

Comment (samuelmi2) [reply validate.sh]: It is fixed with the resolution of the merge conflicts, it was not intended to be committed.

**Comment by techgnosis** [comment]
Closing this PR. A new PR was created that was branched from this master branch so all of this code is already in GCP master.

**Pull request closed without merging**